### PR TITLE
Add a way to directly set HTML font size

### PR DESCRIPTION
**Pull request by shenjackyuanjie**
now can use `<font real_size=20>abc</font>` to set text size directly

### CODE REVIEW

**Comment by shenjackyuanjie** [comment]
sorry, but my respoincy have more commit

File: pyglet/font/win32query.py [hunk 1]
```diff
@@ -340,7 +340,11 @@ def _enum_font_names(logfont, textmetricex, fonttype, param):
     global FONTDB
 
     lf = logfont.contents
-    name = lf.lfFaceName.decode('utf-8')
+    try:
+        name = lf.lfFaceName.decode('utf-8')
+    except UnicodeDecodeError:
+        name = lf.lfFaceName.decode('ANSI')
```
Comment (shenjackyuanjie) [inline pyglet/font/win32query.py:346]: bad windows

**Comment by shenjackyuanjie** [comment]
now it's done!

File: pyglet/font/win32query.py (hunk 1 above)
Comment (shenjackyuanjie) [reply pyglet/font/win32query.py:346]: this is may a way on how to fix #524

**Comment by shenjackyuanjie** [comment]
about the decode of `lfFaceName`
here is a `ttf`` file witch need decode in `ANSI` or `gb2312`
http://gofile.me/4yON4/haluBRI2f

File: pyglet/font/win32query.py (hunk 1 above)
Comment (benmoran56) [reply pyglet/font/win32query.py:346]: If possible, can you move these changes to a new pull request?

Comment (shenjackyuanjie) [reply pyglet/font/win32query.py:346]: > If possible, can you move these changes to a new pull request?

hmmmm，i'll try

**Comment by caffeinepills** [comment]
The `UnicodeDecodeError` portion of this was never separated. https://github.com/pyglet/pyglet/pull/584 Should resolve issue with unicode font names.

Also the file you linked is not available.

**Comment by shenjackyuanjie** [comment]
> 

oh，got it

**Comment by shenjackyuanjie** [comment]
aaaa
see #592  for more!

**Comment by shenjackyuanjie** [comment]
close!

**Pull request closed without merging**